{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "78988eb5_f3547991",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 34
      },
      "lineNbr": 0,
      "author": {
        "id": 1000124
      },
      "writtenOn": "2021-03-17T16:18:41Z",
      "side": 1,
      "message": "Failure in CI unrelated to patch",
      "revId": "76ff17e49be6137e95ae4728c41845bb176a7762",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    }
  ]
}